data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"How strange."`
+				`	"How interesting."`
+			`	"Anyway, I gather you were you after some food?"`
+			label wrapup
+			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
+			`	It's a noisy meal.`
+				decline
+
+#This uses the flag from receiving the Hai translator.
+mission "ZZCU The Meme Wars 1"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector which immediately projects the image of an anthro root vegetable with a sword hacking at some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`
+			branch translator
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost sir but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
+			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
+				decline
+			label translator
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`
```

Comment:
Translator output is weird, but it could be excused by it not being very good at picking up Hai slang.
Do the Hai even have slang?

Reply:
This is how the translator translates Hai, you only otherwise use the translator when the Wanderer's speak Hai to you, because the Hai you encounter generally have learned human standard.

Reply:
I mean that "for the true and square root that is more [significant, meaningful]" is an awkward construction.

Reply:
Ah yes, well in this case consider your translator is essentially transliterating meme text.